web/test/factories.ts

```diff
@@ -129,28 +129,34 @@ export const generateTaxFiling = (overrides: Partial<TaxFiling>): TaxFiling => {
 };
 
 export const generateIndustrySpecificData = (
-  overrides: Partial<IndustrySpecificData>,
-  industry?: Industry,
-  canHavePermanentLocation?: boolean
+  overrides: Partial<IndustrySpecificData>
 ): IndustrySpecificData => {
-  const _industry = industry ?? randomIndustry(canHavePermanentLocation);
   return {
-    liquorLicense: false,
-    requiresCpa: false,
-    homeBasedBusiness: false,
-    cannabisLicenseType: undefined,
-    cannabisMicrobusiness: undefined,
-    constructionRenovationPlan: undefined,
-    providesStaffingService: false,
-    certifiedInteriorDesigner: !!_industry.industryOnboardingQuestions.isCertifiedInteriorDesignerApplicable,
-    realEstateAppraisalManagement: false,
-    carService: undefined,
-    interstateTransport: false,
-    isChildcareForSixOrMore: undefined,
+    liquorLicense: randomElementFromArray([true, false]),
+    requiresCpa: randomElementFromArray([true, false]),
+    homeBasedBusiness: randomElementFromArray([true, false]),
+    cannabisLicenseType: randomElementFromArray(["CONDITIONAL", "ANNUAL"]),
+    cannabisMicrobusiness: randomElementFromArray([true, false]),
+    constructionRenovationPlan: randomElementFromArray([true, false]),
+    providesStaffingService: randomElementFromArray([true, false]),
+    certifiedInteriorDesigner: randomElementFromArray([true, false]),
+    realEstateAppraisalManagement: randomElementFromArray([true, false]),
+    carService: randomElementFromArray(["STANDARD", "HIGH_CAPACITY", "BOTH"]),
+    interstateTransport: randomElementFromArray([true, false]),
+    isChildcareForSixOrMore: randomElementFromArray([true, false]),
```

Comment:
I think that for booleans, something like `!(randomInt() % 2)` like we do in other generators might be a little less verbose